setup.py

```diff
@@ -22,7 +27,7 @@
 
 setup(
     name='pytest-rpc',
-    version='0.13.0',
+    version='1.0.0-dev0',
```

Comment:
Do you want to increment the version in this PR?

Reply:
Never mind I see that it's in a separate commit.

Reply:
BTW, I'm going to keep that bump in the commit history so people know that we had a dev release of the software.